.github/workflows/ci_workflow.yml

```diff
@@ -55,13 +55,40 @@ jobs:
       image: aswf/ci-ocio:${{ matrix.vfx-cy }}
     strategy:
       matrix:
-        build: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+        build: [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12]
```

Comment:
@michdolan I was not able to find a way to specific that all builds must be executed.

Reply:
Are you referring to the new builds not being `required` in the checks? Or just a more streamlined approach instead of listing them all like this?

Reply:
Avoid to list them all i.e. something like ` build: all`

Reply:
Forget my comment.

pybind11 2.4.3 fails to compile with clang 9 & C++17!
But it succeeds with clang 10 & C++17 so I disabled build 11 & 12 for now.

Reply:
Because of the way GH Action matrix is currently designed, this index listing workaround was my way of supporting a variety of builds without exponentially increasing jobs.  This essentially replicates the Azure Pipelines matrix behavior, instead of GHA's current logic, which does:

matrix:
  a: [1, 2]
  b: [1, 2]

will result in jobs:
a1b1
a2b1
a1b2
a2b2

So with our number of matrix variables the job count becomes unmanagemable. Correcting this is an active GHA request, so hopefully we can improve it later.